### ISSUE TITLE
Keyboard nav doesn't always work in Equation combo box

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/fourier-making-waves#195]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/711. Seen with keyboard nav
When using the arrow keys to move through the options in the equation combo box on the first screen, only some of the inputs take. Many don't seem to register. The other combo boxes on that page don't seem to do this.

**Visuals**

https://user-images.githubusercontent.com/41024075/135163737-5ef7b6b6-7f08-4097-a26f-06f599510039.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Fourier: Making Waves‬
URL: https://phet-dev.colorado.edu/html/fourier-making-waves/1.0.0-rc.1/phet/fourier-making-waves_all_phet.html
Version: 1.0.0-rc.1 2021-09-28 15:44:23 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.61 Safari/537.36
Language: en-US
Window: 1280x657
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

_pixelzoom EDIT: replaced "dropdown box" with "combo box" throughout._

**Comment by pixelzoom** [comment]
The Equations combo box has a dynamic list.   Items in the list are visible based on the domain (Function of).  My guess is that combo box is not handling focus traversal properly, and is trying to traverse through items that are invisible. 

@jessegreenberg appears to be the responsible dev for focus traversal in ComboBoxListBox, so I'll start by assigning to him.  Are there changes that need to be made to support invisible items?

**Comment by pixelzoom** [comment]
I investigated the general problem, and it's indeed a bug in ComboBoxListBox. Tracking in https://github.com/phetsims/sun/issues/720, and putting this issue on-hold until it's fixed.

**Comment by pixelzoom** [comment]
phetsims/sun#720 is fixed. I'll need to patch https://github.com/phetsims/sun/commit/8248b7bd3e73a08b5376f6721a2d68f77a87afcd to into the 1.0 branch.

**Comment by pixelzoom** [comment]
https://github.com/phetsims/sun/commit/8248b7bd3e73a08b5376f6721a2d68f77a87afcd was cherry-picked into a new branch of sun, and patched into Fourier 1.0.

Ready for verification in the next RC.

**Comment by Nancy-Salpepi** [comment]
Keyboard nav in Equations Combo Box is working properly now (verified using above steps).
